### ISSUE TITLE
can nougat utilize multi GPUs?

### ISSUE THREAD

**Issue by jackgreen12345** [facebookresearch/nougat#63]
in nougat's options, we don't need to allocate GPUs. we have multiple GPUs in the server, can nougat use multi GPUs in one task？
if nougat can only use one GPU in one task, can we setup serveral tasks which run one task with one GPU?

**Comment by lukas-blecher** [comment]
No, inference only works on 1 GPU. You'll have to split the load manually, as you suggested.

**Comment by marwinsteiner** [comment]
> in nougat's options, we don't need to allocate GPUs. we have multiple GPUs in the server, can nougat use multi GPUs in one task？ if nougat can only use one GPU in one task, can we run serveral tasks which one task run with one GPU?

have you already implemented that? Would be interested in seeing your approach to this
